File: python3/iplocation.py
#!/usr/bin/python3
import urllib.request, json

access_token = "changeme" #Your justreqit access_token...

def ip_location(ip):
	api_url = "https://justreqit.com/api/ipv4/%s/?access_token=%s" % (ip, access_token)
	response = json.loads(urllib.request.urlopen(api_url).read())
	if (response["status"] == "success"):
		return "%s, %s" % (response["results"]["city"], response["results"]["state"])
	else:
		return None

#example 
print(ip_location("140.82.112.3"))

File: php/iplocation.php
<?php

//Your justreqit.com access_token
$access_token = 'changeme';

function ip_location($ip) {
	global $access_token;
	$api_url = 'https://justreqit.com/api/ipv4/'.$ip.'/?access_token='.$access_token;
	$response = json_decode(file_get_contents($api_url));
	if ($response->status == "success"){
		return $response->results->city.', '.$response->results->state;
	} else {
		return null;
	}
}


echo ip_location("140.82.112.3");


?>